iModelCore/ECDb/Tests/NonPublished/ECSqlStatementTests.cpp

```diff
@@ -9017,6 +9017,104 @@ TEST_F(ECSqlStatementTestFixture, OrderBy)
     ASSERT_EQ(0, strcmp(lastName, "Da Vinci"));
     }
 
+
+//---------------------------------------------------------------------------------------
+// @bsimethod
+//---------------------------------------------------------------------------------------
+
+TEST_F(ECSqlStatementTestFixture, NullsOrdering)
+{
+    // Create StartupCompany
+    ASSERT_EQ(BentleyStatus::SUCCESS, SetupECDb("StartupCompany.ecdb", SchemaItem::CreateForFile("StartupCompany.02.00.00.ecschema.xml")));
+
+    auto insertPerson = [](ECDbCR ecdb, ECClassCR ecClass, Utf8CP firstName, Utf8CP lastName) {
+        IECInstancePtr ecInstance = ecClass.GetDefaultStandaloneEnabler()->CreateInstance(0);

```

Comment:
Better use simple insert statements for this instead of the instance inserter.
Like this:
ASSERT_ECSQL(m_ecdb, ECSqlStatus::Success, BE_SQLITE_DONE, "INSERT INTO TestSchema.A (MyProperty) VALUES ('A')");

It's a single line, should shrink the test code significantly

Reply:
This macro is not included in ECSqlStatementTets.cpp, which is why I am using the instance inserter.

Reply:
Using the macro or writing out the code to insert using ECSql would be shorter and easier to read.  Switch to an ECSql INSERT method

Reply:
I have included the macro and  made the required changes in the test.